### PR TITLE
Updated federation docs

### PR DESCRIPTION
**Pull request by michalxo**
Last bit - I have also reviewed `amqp-broker-connections.adoc` as I was learning/reading it. 
No more new additions from me. Just fixes.

### CODE REVIEW

File: docs/user-manual/federation-address.adoc [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
+Address federation is similar to full multicast over the connected brokers. Every message sent to address on `Broker-A` will be delivered to every queue on that broker, but also will be delivered to `Broker-B` and all their attached queues.
 
 Address federation dynamically links to other addresses in upstream or downstream brokers.
-It automatically creates a queue on the remote address for itself,  to which then it consumes, copying to the local address, as though they were published directly to it.
+It automatically creates a queue on the remote address for itself, which it then consumes and copies to the local address, as if they were published directly to it.
 
-The upstream brokers do not need to be reconfigured or the address, simply permissions to the address need to be  given to the address for the downstream broker.
-Similarly the same applies for downstream configurations.
+Upstream brokers or addresses do not need to be reconfigured. Just add needed permissions to the address for the downstream broker. Same applies for downstream configurations.
```
Comment (gtully) [inline docs/user-manual/federation-address.adoc:10]: "the needed"

File: docs/user-manual/federation-address.adoc [hunk 2]
```diff
@@ -112,96 +111,97 @@ Sample Address Federation setup:
...
+The address-match pattern to include addresses. Multiple of these can be set. If none are set all addresses are matched.
+
 exclude::
-The address-match pattern to exclude addresses.
-Multiple of these can be set.
+The address-match pattern to exclude addresses. Multiple of these can be set.
+
 max-hops::
-The number of hops that a message can have made for it to be federated, see <<topology-patterns,Topology Patterns>> above for more details.
+The number of maximum hops that message can perform across federated addresses. See <<topology-patterns,Topology Patterns>> above for details.
```
Comment (tabish121) [inline docs/user-manual/federation-address.adoc:133]: "The maximum number of hops..."

File: docs/user-manual/federation-address.adoc [hunk 3]
```diff
@@ -112,96 +111,97 @@ Sample Address Federation setup:
...
-Now look at all the `transformer` parameters in turn, in order of priority:
+==== Priority ordered transformer parameters
 
 name::
-This must be a unique name in the server, and is used to ref the transformer in `address-policy` and `queue-policy`
+Unique name in the server used to reference the transformer in `address-policy` and `queue-policy`.
 
 transformer-class-name::
-An optional transformer-class-name can be specified.
+An optional `transformer-class-name` can be specified.
```
Comment (gemmellr) [inline docs/user-manual/federation-address.adoc:160]: I dont really see a point in immediately highlighting the name of the attribute, which is already much more significantly bolded / highlighted in the overall attribute list anyway. If changing it, maybe just remove the hyphens instead? Or combine the 'optional' bit into the similar sentence that follows.

File: docs/user-manual/federation-address.adoc [hunk 4]
```diff
@@ -83,19 +82,19 @@ Sample Address Federation setup:
         <upstream name="eu-east-1">
            <static-connectors>
               <connector-ref>eu-east-connector1</connector-ref>
-              <connector-ref>eu-east-connector1</connector-ref>
+              <connector-ref>eu-east-connector2</connector-ref>
```
Comment (gemmellr) [inline docs/user-manual/federation-address.adoc:85]: You have changed connector refs in almost every config example in the doc, fixing some apparent bugs within themselves...however those config examples all seem to remain inconsistent with each other in terms of refs. Varying connector name formats like "connector1", "eu-east-connector2", and "eu-east-1-connector" all remain. Should those that seem related to an overall geographic example perhaps actually be aligned to use the same naming format?

File: docs/user-manual/federation-address.adoc [hunk 5]
```diff
@@ -112,96 +111,97 @@ Sample Address Federation setup:
...
+In the above setup, downstream broker `eu-north-1` is configured to connect to two upstream brokers `eu-east-1` and `eu-west-1`. Credentials used for both connections to brokers in this sample are shared.
+Should they be different per upstream, you can alter credentials at the upstream level.
 
-Both upstreams are configured with the same address-policy `news-address-federation`, that is selecting addresses which match any of the include criteria, but will exclude anything that starts `queue.news.sport`.
+Both upstreams are configured with the same address-policy `news-address-federation`, that is selecting addresses which match any of the include criteria and exclude anything that starts `queue.news.sport`.
 
 *It is important that federation name is globally unique.*
 
-Let's take a look at all the `address-policy` parameters in turn, in order of priority.
+==== Priority ordered address-policy parameters
```
[the remark this answers is not in the document]
Comment (michalxo) [reply docs/user-manual/federation-address.adoc:121]: Agree with you. I will remove that priority order. Neither I like it :)

File: docs/user-manual/federation-address.adoc (hunk 3 above)
Comment (michalxo) [reply docs/user-manual/federation-address.adoc:160]: Agree. I will remove whole sentence. It follows later
`If specified, then the ...`

File: docs/user-manual/federation-address.adoc (hunk 4 above)
Comment (michalxo) [reply docs/user-manual/federation-address.adoc:85]: TBH I don't know. I consider xml excerpts as isolated examples. 
If I am wrong, probably original author/s should fix appropriately?

**Comment by jbertram** [comment]
I squashed the commits, cleaned up a few things, created a Jira, updated the commit message, and merged.

**Pull request closed without merging**